content/installation/setup/auth/SSO.md

```diff
@@ -99,6 +99,43 @@ Once connected, you can return to the **SSO** tab to view and edit your settings
 
 <a href="assets/images/Sso-org-settings-connected.png" rel="lightbox" title="Edit or revert SSO settings in Organization Settings"><img class="thumbnail" src="assets/images/Sso-org-settings-connected.png"/></a>
 
+## Implement Group Membership
+
+For EOP and SaaS accounts with SSO enabled, a system administrator can automatically add users to organizational groups in Contrast according to group membership configured in the IdP. To set this up for your organization, you must update your SAML configuration in your IdP provider and your **SSO** settings in Contrast UI. 
+
+### Configuration 
+
+#### SAML 
+
+Contrast expects group information to be sent by the IdP as a multi-value attribute `contrast_group` in your SAML configuration.
```

Comment:
@ubermitchy I thought the field name was `contrast_groups` - as in plural. Can we double check this?